data/pathfinder/paizo/player_companion/blood_of_the_moon/botm_races.lst

```diff
@@ -2,7 +2,7 @@
 
 
 # Race Name	Starting Feats	Size		Move			Modify VAR				Ability								Nb Legs	Nb Hands	Main Race Type	Race Subtype			Source Page
-Skinwalker	STARTFEATS:1	SIZE:M	MOVE:Walk,30	BONUS:VAR|HasLowlightVision|1	ABILITY:Internal|AUTOMATIC|Race Traits ~ Skinwalker	LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Shapechanger	SOURCEPAGE:p.7
+Skinwalker	STARTFEATS:1	SIZE:M	MOVE:Walk,30	LANGBONUS:ALL	BONUS:VAR|HasLowlightVision|1	AUTO:LANG|Common	ABILITY:Internal|AUTOMATIC|Race Traits ~ Skinwalker	LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Shapechanger	TYPE:Base.PC	SOURCEPAGE:p.7	FACT:IsPC|True
```

Comment:
This is not following core examples. Languages are granted via abilities and templates.